BackEnd/src/routes/api/task.js

```diff
@@ -67,7 +67,7 @@ createRoutes([
                     if (value != undefined) options[key] = value;
                 }
 
-                editTask(args.taskId, options);
+                await editTask(args.taskId, options);
```

Comment:
Der Refactor hier hat bestimmt nichts mit der Schnittstellenanpassung zu tun, oder? Aber wenn du das Ergebnis schon abwartest, dann solltest du vermutlich auch auswerten, ob die Anfrage erfolgreich war und nicht erst das (evtl negative) Ergebnis abwarten um dann trotzdem 200 zurückzugeben, oder?

Reply:
Hat er tatsächlich nicht, ich würds trotzdem hier gerne mit reinschmuggeln, weil es nur sone Kleinigkeit ist und ich dafür nicht extra nen Issue anlegen will.

Falls in der Anfrage etwas schiefläuft, wird ein Fehler geworfen, dieser wird durchs try catch gefangen und mit nem 400er Code zurückgegeben. Wäre das await nicht da, würde nur der Promise erstellt werden, dann wäre die 200er Antwort zurückgegangen. Falls dann mit der DB etwas schiefgegangen wäre, hätte der Client bereits seine 200er Antwort bekommen.

![grafik](https://user-images.githubusercontent.com/69909446/110244352-9fd32380-7f5e-11eb-904b-bb68b5027011.png)
=>

![grafik](https://user-images.githubusercontent.com/69909446/110244374-bb3e2e80-7f5e-11eb-8246-55278d899cef.png)